src/net.cpp

```diff
@@ -2324,7 +2324,7 @@ void CNode::AskFor(const CInv& inv)
 
 void CNode::BeginMessage(const char* pszCommand) EXCLUSIVE_LOCK_FUNCTION(cs_vSend)
 {
-    ENTER_CRITICAL_SECTION(cs_vSend);
+    ENTER_CRITICAL_SECTION(cs_vSend)
```

Comment:
If there were more uses of `{ENTER,LEAVE}_CRITICAL_SECTION` then I'd suggest altering the macros to avoid the warning, but it turns out there are only 8 uses of either macro in the whole codebase.

Reply:
I edited the macros in other places where it made sense (e.g., https://github.com/zcash/zcash/pull/6636/commits/52f41054df1e2cb5dd2d8e3f08570f25924125f4#diff-b1924661640b70276005001174b3b3640f02be7232bb8d9a1b9518dde32f8055R189). But here (and for `ADD_SERIALIZE_METHODS` in #6641) the structure of the macro made it non-obvious how I could make that change.